pkg/commands/git.go

```diff
@@ -274,40 +277,42 @@ func (c *GitCommand) AbortMerge() error {
 	return c.OSCommand.RunCommand("git merge --abort")
 }
 
-// UsingGpg tells us whether the user has gpg enabled so that we can know
+// usingGpg tells us whether the user has gpg enabled so that we can know
 // whether we need to run a subprocess to allow them to enter their password
-func (c *GitCommand) UsingGpg() bool {
-	gpgsign, _ := gitconfig.Global("commit.gpgsign")
+func (c *GitCommand) usingGpg() bool {
+	gpgsign, _ := c.getGlobalGitConfig("commit.gpgsign")
```

Comment:
This is really a criticism of the earlier code, but can we swap the global and the local check? It makes sense to use the local value and then default to the global one

Reply:
Yes sorry I forgot about your comment earlier and it got hidden.